.circleci/config.yml

```diff
@@ -0,0 +1,95 @@
+# Python CircleCI 2.0 configuration file
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+version: 2
+jobs:
+  build:
+    docker:
+      # CircleCI maintains a library of pre-built images documented at
+      # https://circleci.com/docs/2.0/circleci-images/
+      # use `-browsers` prefix for selenium tests, e.g. `<image_name>-browsers`
+
+      # Python
+      - image: circleci/python:2.7.13
+        environment:
+          TZ: America/New_York
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+          - fec-eregs-dependencies-{{ checksum "requirements.txt" }}-{{ checksum "requirements_dev.txt" }}-{{ checksum "requirements_test.txt" }}-{{ checksum "package.json" }}
```

Comment:
Are the `dev` dependencies required?  If not, we could remove it from this key name.

Reply:
They are- I don't think we should have a dev dependencies file if you need them for deploy but I did not dig into it because we need to refactor with the upgrade anyway

Reply:
Fair enough, just wanted to be sure!